packages/forms-web-app/src/dynamic-forms/journey-response.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Defines a response to a journey, a set of Answers to the questions
+ * @class
+ */
+class JourneyResponse {
+	/**
+	 * @type {string} - the unique reference id used in the url for the journey
+	 */
+	referenceId;
```

Comment:
I think you'll also need a `formId` as the `referenceId` on its own won't necessarily be unique (as we may have multiple forms per appeal e.g. LPA questionnaire, final comments, LPA statements, etc.).

Reply:
Are you thinking just an arbitrary string e.g. has-questionnaire?

Reply:
have done this